distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -90,3 +90,28 @@ MOVE;$OPENHAB_USERDATA/config/org/eclipse/smarthome/storage/json.config;$OPENHAB
 REPLACE;org.eclipse.smarthome.storage.json;org.openhab.storage.json;$OPENHAB_USERDATA/config/org/openhab/storage/json.config
 MOVE;$OPENHAB_USERDATA/config/org/eclipse/smarthome/storage/mapdb.config;$OPENHAB_USERDATA/config/org/openhab/storage/mapdb.config
 REPLACE;org.eclipse.smarthome.storage.mapdb;org.openhab.storage.mapdb;$OPENHAB_USERDATA/config/org/openhab/storage/json.config
+
+REPLACE;org.eclipse.smarthome.core;org.openhab.core;$OPENHAB_USERDATA/jsondb/automation_rules.json
+REPLACE;org.eclipse.smarthome;org.openhab.core;$OPENHAB_USERDATA/jsondb/automation_rules.json
+
+MOVE;$OPENHAB_USERDATA/jsondb/org.eclipse.smarthome.config.discovery.DiscoveryResult.json;$OPENHAB_USERDATA/jsondb/org.openhab.core.config.discovery.DiscoveryResult.json
+REPLACE;org.eclipse.smarthome.core;org.openhab.core;$OPENHAB_USERDATA/jsondb/org.openhab.core.config.discovery.DiscoveryResult.json
+REPLACE;org.eclipse.smarthome;org.openhab.core;$OPENHAB_USERDATA/jsondb/org.openhab.core.config.discovery.DiscoveryResult.json
+
+MOVE;$OPENHAB_USERDATA/jsondb/org.eclipse.smarthome.core.items.Item.json;$OPENHAB_USERDATA/jsondb/org.openhab.core.items.Item.json
+REPLACE;org.eclipse.smarthome.core;org.openhab.core;$OPENHAB_USERDATA/jsondb/org.openhab.core.items.Item.json
+REPLACE;org.eclipse.smarthome;org.openhab.core;$OPENHAB_USERDATA/jsondb/org.openhab.core.items.Item.json
+
+MOVE;$OPENHAB_USERDATA/jsondb/org.eclipse.smarthome.core.items.Metadata.json;$OPENHAB_USERDATA/jsondb/org.openhab.core.items.Metadata.json
+REPLACE;org.eclipse.smarthome.core;org.openhab.core;$OPENHAB_USERDATA/jsondb/org.openhab.core.items.Metadata.json
+REPLACE;org.eclipse.smarthome;org.openhab.core;$OPENHAB_USERDATA/jsondb/org.openhab.core.items.Metadata.json
+
+MOVE;$OPENHAB_USERDATA/jsondb/org.eclipse.smarthome.core.thing.link.ItemChannelLink.json;$OPENHAB_USERDATA/jsondb/org.openhab.core.thing.link.ItemChannelLink.json
+REPLACE;org.eclipse.smarthome.core;org.openhab.core;$OPENHAB_USERDATA/jsondb/org.openhab.core.thing.link.ItemChannelLink.json
+REPLACE;org.eclipse.smarthome;org.openhab.core;$OPENHAB_USERDATA/jsondb/org.openhab.core.thing.link.ItemChannelLink.json
+
+MOVE;$OPENHAB_USERDATA/jsondb/org.eclipse.smarthome.core.thing.Thing.json;$OPENHAB_USERDATA/jsondb/org.openhab.core.thing.Thing.json
+REPLACE;org.eclipse.smarthome.core;org.openhab.core;$OPENHAB_USERDATA/jsondb/org.openhab.core.thing.Thing.json
+REPLACE;org.eclipse.smarthome;org.openhab.core;$OPENHAB_USERDATA/jsondb/org.openhab.core.thing.Thing.json
+
+DELETE;$OPENHAB_USERDATA/jsondb/org.eclipse.smarthome.core.thing.link.ItemThingLink.json
```

Comment:
These files have been unused for quite some time now (see https://github.com/eclipse/smarthome/pull/4102).